src/lib/input/input.ts

```diff
@@ -229,6 +229,10 @@ export class MdInput implements ControlValueAccessor, AfterContentInit, OnChange
 
   @ViewChild('input') _inputElement: ElementRef;
 
+  public elementType: 'input' | 'textarea' = undefined;
+
+  constructor(private _elRef: ElementRef) { }
```

Comment:
If you move the setting of `elementType` here, you don't need the `private` and can just call it `elementRef`.